providers/base.ts

```diff
@@ -216,17 +216,23 @@ export abstract class ReleaseLookup<Provider extends MetadataProvider, RawReleas
 			}
 
 			const releaseType = this.provider.entityTypeMap['release'];
-			if (entity.type !== releaseType) {
+			if (
+				!(
+					Array.isArray(releaseType) ? releaseType.includes(entity.type) : entity.type === releaseType
+				)
+			) {
 				throw new ProviderError(this.provider.name, `${specifier} is not a release URL`);
 			}
-			this.lookup = { method: 'id', value: entity.id };
+			this.id = entity.id;
```

Comment:
I assigned the `id` here so that I could access it in `getRawRelease`. Is the assignment in `convertRawRelease` still needed?